osu.Game/Overlays/BeatmapSet/MetadataSection.cs

```diff
@@ -53,31 +70,31 @@ public MetadataSection(MetadataType type, Action<string> searchAction = null)
                         AutoSizeAxes = Axes.Y,
                         Child = new OsuSpriteText
                         {
-                            Text = this.type.GetLocalisableDescription(),
+                            Text = type.GetLocalisableDescription(),
                             Font = OsuFont.GetFont(weight: FontWeight.Bold, size: 14),
                         },
                     },
                 },
             };
         }
 
-        public string Text
+        public virtual T Text
```

Comment:
This is a bit strange as it is called `Text` but... isn't necessarily text? Can we rename to `Metadata` or something?

Reply:
I actually did rename this to `Metadata` when I was refactoring but I wanted to avoid renaming things / having it be renamed again if the name change didn't cut it.